server/channels/app/properties/property_group.go

```diff
@@ -0,0 +1,16 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+package properties
+
+import (
+	"github.com/mattermost/mattermost/server/public/model"
+)
+
+func (ps *PropertyService) RegisterPropertyGroup(name string) (*model.PropertyGroup, error) {
+	return ps.groupStore.Register(name)
+}
```

Comment:
Shouldn't we at least have a delete option? What happens if I register a property group with e.g. a typo?